joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonAstVisitorHelpers.scala

```diff
@@ -67,8 +68,13 @@ trait PythonAstVisitorHelpers { this: PythonAstVisitor =>
           createIdentifierNode("import", Load, lineAndCol),
           "import",
           lineAndCol,
-          nodeBuilder.stringLiteralNode(from, lineAndCol) ::
-            nodeBuilder.stringLiteralNode(alias.name, lineAndCol) :: Nil,
+          Seq(
+            nodeBuilder.stringLiteralNode(from, lineAndCol),
+            nodeBuilder.stringLiteralNode(alias.name, lineAndCol)
+          ) ++ (alias.asName match {
+            case Some(aliasName) => Seq(nodeBuilder.stringLiteralNode(aliasName, lineAndCol))
```

Comment:
Here we add an optional third argument, which aims to be the alias name. Should we make this a special named argument?

Reply:
What do you mean by "special named"?